### PR TITLE
update CJKRadicals & EqUIdeo

### PR DESCRIPTION
**Pull request by markusicu**
[[179-A22](https://www.unicode.org/cgi-bin/GetL2Ref.pl?179-A22)] Action Item for Markus Scherer, PAG: Update the UCD’s CJKRadicals.txt data file, based on PRI #483 feedback [ID20240328172102] and Section 10 of document [L2/24-067](https://www.unicode.org/cgi-bin/GetMatchingDocs.pl?L2/24-067) and its attached CJKRadicals-20240405.txt data file, for Unicode Version 16.0.

[[179-A71](https://www.unicode.org/cgi-bin/GetL2Ref.pl?179-A71)] Action Item for Markus Scherer, PAG: Update the UCD’s EquivalentUnifiedIdeograph.txt data file to append two lines for U+31E4 and U+31E5, based on document [L2/24-044](https://www.unicode.org/cgi-bin/GetMatchingDocs.pl?L2/24-044) and Section 32 of document [L2/24-067](https://www.unicode.org/cgi-bin/GetMatchingDocs.pl?L2/24-067) and its attached EquivalentUnifiedIdeograph-20240405.txt data file, for Unicode Version 16.0.

### CODE REVIEW

**Review by kenlunde — APPROVED** [review]

**Comment by markusicu** [comment]
Can someone else please approve as well? KenL does not (yet?) have write access here, so his approval is good but not sufficient :-/

**Review by eggrobin — APPROVED** [review]

**Pull request merged**
